### PR TITLE
Mrc-3240 Include system info in logs

### PR DESCRIPTION
**Pull request by LekanAnni**
## Description

PR uses application url in structured log to differentiate different profiles such as staging, production or dev. 

## Type of version change

Minor 

## Checklist

- [x] I have incremented version number, or version needs no increment
- [x] The build passed successfully, or failed because of ADR tests

### CODE REVIEW

**Review by Nicolas-Dolan — APPROVED** [review]

**Review by EmmaLRussell — COMMENTED** [review]
Code looks fine to me. `profile` seems like a slightly weird prop name now it's logging the whole base url (not 'dev' or 'prod' as was suggested in the ticket). Would just `url` be more accurate?

**Comment by LekanAnni** [comment]
> Code looks fine to me. `profile` seems like a slightly weird prop name now it's logging the whole base url (not 'dev' or 'prod' as was suggested in the ticket). Would just `url` be more accurate?

it could be profileUrl, the whole idea is to differentiate deployments, which we can achieve using this approach. Using dev, prod, staging or whatever would mean we have to resolve all deployments manually.

**Comment by EmmaLRussell** [comment]
> > Code looks fine to me. `profile` seems like a slightly weird prop name now it's logging the whole base url (not 'dev' or 'prod' as was suggested in the ticket). Would just `url` be more accurate?
> 
> it could be profileUrl, the whole idea is to differentiate deployments, which we can achieve using this approach. Using dev, prod, staging or whatever would mean we have to resolve all deployments manually.

`profileUrl` sounds good

File: NEWS.md [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
 * Include system info in logs
 
+# hint 2.14.0
+
+* Add Auth0 login method
+
+# hint 2.13.0
+
+* Update hint to work with options schema changes
+>>>>>>> master
```
Comment (EmmaLRussell) [inline NEWS.md:12]: rogue merge artefact

**Review by EmmaLRussell — APPROVED** [review]
Thanks - just needs leftover merge artefact removing,..

**Pull request merged**